### DISCUSSION TITLE
Is KVS downgrading still viable

### DISCUSSION THREAD

**Discussion by Darcii** [3kh0/ext-remover#1266] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hello, I've got a drive with version 1.25 on it, and I don't believe you can downgrade any further. I want to use the KVS exploit, and I found a public site with all the shims and files I might need. Since they have blocked downgrading, I'm wondering if it's still possible to downgrade via the kernel. If so, I could theoretically shim it and then use FakeMurk right?

**Comment by Darcii** [comment]
Ok wow unrelated however I was looking through my schools policy allowed list and looks like someone convinced the admins to unblock tampermonkey because its on the allowlist... Wtf lmao

**Reply by HelloIloveGimkit2** [reply]
lol

**Comment by HelloIloveGimkit2** [comment, marked as answer]
also yes I'm pretty sure it's still possible.
But you're better off just using br1ck (use my fork cuz it's got something about skipping the 5 min wait timer for devmode)
[br1ck](https://sk1mmer.vercel.app)

**Reply by Darcii** [reply]
Well I believe my dedede laptop is keyrolled so I would have to buy a suzy qable n I dont wanna do that lmao

**Reply by HelloIloveGimkit2** [reply]
why do you believe so

**Reply by Darcii** [reply]
> why do you believe so

isnt a keyrolled device a enterprise enrolled device? Meaning all are keyrolled if not can you explain it to me

**Reply by HelloIloveGimkit2** [reply]
no cuz mine was that's not what it means contact @Byteeeeeeee about that cuz idk but it doesn't mean that ur key rolled if it's enterprise enrolled cuz mine was and I'm not
